tests/components/ring/test_sensor.py

```diff
@@ -132,7 +133,11 @@ async def test_history_sensor(
     expected_value,
 ) -> None:
     """Test the Ring sensors."""
-    await setup_platform(hass, "sensor")
+    with patch(
+        "homeassistant.components.ring.sensor.async_check_create_deprecated",
+        return_value=True,
+    ):
```

Comment:
Please avoid patching the integration itself in tests

Reply:
Done